adding sql file

tristan-nunnally opened the pull request.

tristan-nunnally committed:
adding sql file

drewbanin commented on refactor.sql:
```diff
@@ -1,3 +1,5 @@
+simple changeX
```
Hm, what's going on here?

tristan-nunnally commented:
@drewbanin FYI and thanks!

drewbanin reviewed (changes requested):
@tristan-nunnally nice PR! I have a couple of questions about the code changes here:
1) my first comment
2) my second comment
3) finally, a third comment about these code changes

Can you X and Y? Happy to meet up to discuss!

tristan-nunnally committed:
Removed bogus SQL.

@drewbanin thanks for catching that! I updated the file to remove  "simple changeX". Not sure where that came from 😅 

Can you review once more?

drewbanin commented:
LGTM! 🚢 it :)
